connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SessionHolder.scala

```diff
@@ -44,6 +45,8 @@ case class SessionHolder(userId: String, sessionId: String, session: SparkSessio
   val executions: ConcurrentMap[String, ExecuteHolder] =
     new ConcurrentHashMap[String, ExecuteHolder]()
 
+  val events: SessionEvents = SessionEvents(this, new SystemClock())
```

Comment:
naming nit: maybe eventManager: SessionEventsManager?
"events" suggests that it buffers or stores the events, but it only manages getting them to the listener bus.